src/client/auth/actions.js

```diff
@@ -33,14 +33,23 @@ function validateForm(fields) {
 }
 
 function validateCredentials(fields) {
-  // Simulate lengthy async action.
   return new Promise((resolve, reject) => {
-    setTimeout(() => {
-      if (fields.password === 'pass1')
+
+    var xhr = new XMLHttpRequest();
```

Comment:
The only problem currently is this bug in Chromium: https://code.google.com/p/chromium/issues/detail?id=457484

You cannot see the POST body in Dev Tools.